tests/Operation/FindFunctionalTest.php

```diff
@@ -123,6 +125,45 @@ public function provideTypeMapOptionsAndExpectedDocuments()
         ];
     }
 
+    public function testMaxAwaitTimeMS()
+    {
+        $maxAwaitTimeMS = 10;
+
+        // Create a capped collection.
+        $databaseName = $this->getDatabaseName();
+        $cappedCollectionName = $this->getCollectionName() . '.capped';
+        $cappedCollectionOptions = [
+            'capped' => true,
+            'max' => 100,
+            'size' => 1048576,
+        ];
+
+        $operation = new CreateCollection($databaseName, $cappedCollectionName, $cappedCollectionOptions);
+        $operation->execute($this->getPrimaryServer());
+
+        // Insert documents into the capped collection.
+        $cappedNamespace = $databaseName . '.' . $cappedCollectionName;
+        $bulkWrite = new BulkWrite(['ordered' => true]);
+        $bulkWrite->insert(['_id' => 1]);
+        $result = $this->manager->executeBulkWrite($cappedNamespace, $bulkWrite);
```

Comment:
If you end up using the default collection name (without a ".capped" suffix), we can also replace this local variable with `$this->getNamespace()`.